recipes/grpc/all/conanfile.py

```diff
@@ -79,9 +79,6 @@ def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
             tools.check_min_cppstd(self, 11)
 
-        if tools.is_apple_os(self.settings.os) and "arm" in self.settings.arch:
-            raise ConanInvalidConfiguration("gRPC is not supported on M1 Mac due to limitations in protobuf support.")
```

Comment:
If it is because `protobuf`, does `protobuf` raises `ConanInvalidConfiguration` for this configuration? If so, it should already invalidate the configuration for consumers.